src/device-registry/controllers/create-site.js

```diff
@@ -649,14 +647,10 @@ const manageSite = {
       if (isEmpty(tenant)) {
         tenant = constants.DEFAULT_TENANT || "airqo";
       }
-      let filter = generateFilter.sites(req);
-      filter["category"] = "summary";
-      let responseFromListSites = await createSiteUtil.list({
-        tenant,
-        filter,
-        limit,
-        skip,
-      });
+      let request = Object.assign({}, req);
+      request.query.tenant = tenant;
+      request.category = "summary";
+      const responseFromListSites = await createSiteUtil.list(request);
```

Comment:
this was the main part which resolved the issue of listing Sites and eventually resolved the STAGING map issues. @BenjaminSsempala 

Recent refactoring has changed how the util works but forgot to adjust the controller function accordingly